privacyidea/api/before_after.py

```diff
@@ -85,6 +85,19 @@ def teardown_request(exc):
     log.debug(u"End handling of request {!r}".format(request.full_path))
 
 
+# NOTE: This can be commented in to debug SQL pooling issues
+#@token_blueprint.before_app_request
+#def log_pools():
```

Comment:
I think we would only need to comment out the decorator?!

However, then we would have untested code! ;-)